blocks/banner/banner.css

```diff
@@ -0,0 +1,180 @@
+.banner .banner-picture {
+  position: relative;
+  height: 375px;
+  padding: 0 16px;
+  max-width: 1200px;
+  background-size: cover;
+  background-position: center;
+}
+
+.banner .banner-element {
+  padding-bottom: 25px;
+  background-size: cover;
+  background-position: center;
+}
+
+.banner .spotlight-link {
+  color: inherit;
+}
+
+.banner .spotlight-text .title {
+  font-size: 42px;
+  font-weight: 900;
+  line-height: 1em;
+  padding-bottom: 20px;
+  font-family: var(--heading-font-family);
+}
+
+.banner .spotlight-text .subtitle {
+  font-size: 36px;
+  font-weight: 900;
+  line-height: 1em;
+  padding-bottom: 20px;
+  font-family: var(--heading-font-family);
+}
+
+.banner .spotlight-button {
+  background-color: var(--btn-bg-color-3);
+  color: var(--background-color);
+  margin-bottom: 20px;
+}
+
+.banner .spotlight-form {
+  height: 30px;
+}
+
+.banner .spotlight-input {
+  margin: 0%;
+  height: 30px;
+  box-sizing: border-box;
+  padding: 5px 10px;
+  font-size: var(--body-font-size-xs);
+  line-height: 1.5;
+  border: 2px solid var(--dark-color);
+}
+
+.banner .spotlight-input:focus {
+  border-color: var(--input-focus-border);
+  outline: 0;
+  box-shadow: inset 0 1px 1px rgba(0 0 0 / 7.5%), 0 0 8px rgba(102 175 233 /60%);
+}
+
+.banner .spotlight-form-button {
+  background-color: var(--yellow);
+  color: var(--dark-blue);
+  font-size: var(--body-font-size-xs);
+  box-sizing: border-box;
+  border-radius: 0 4px 4px 0;
+  max-height: 30px;
+}
+
+.banner .spotlight-circle-holder {
+  max-height: 375px;
+  box-sizing: border-box;
+  overflow: hidden;
+  position: relative;
+}
+
+.banner .spotlight-circle-holder .spotlight-circle {
+  border-radius: 288px;
+  height: 530px;
+  margin: auto;
+  border: 10px solid white;
+  transform: translateY(-15%);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.banner .spotlight-text {
+  text-align: center;
+  padding: 0 10px;
+  margin: auto;
+}
+
+.banner .mobile-bannertext-holder {
+  padding: 20px;
+}
+
+.banner-wrapper:nth-child(odd) .spotlight-text .title {
+  color: var(--link-color);
+}
+
+.banner-wrapper:nth-child(even) .spotlight-text .title {
```

Comment:
The dom doesn't look right here: 
![image](https://github.com/aemsites/hubblehomes-com/assets/6395150/7432cb04-913b-41eb-844e-b7c8a185b59d)
The spotlight-text is applied to the div and probably shouldn't be.

Reply:
The `spotlight-text class` is meant for the container holding all the text in the banner. However, it seems to have been mistakenly applied to the h2 and h3 elements too. I'll fix that up.